src/d3a/models/const.py

```diff
@@ -137,3 +139,16 @@ class BalancingSettings:
         OFFER_SUPPLY_RATIO = 0.1
         # Adds flexible load support.
         FLEXIBLE_LOADS_SUPPORT = True
+
+
+class GlobalConfig:
```

Comment:
Is there any particular advantage to not make it a subclass of ConstSettings

Reply:
It really is a matter of preference I think.
I see it as a singleton for the global simulation configuration, that is set for for each simulation run. The others are constants and default values for the strategies. Additionally, these parameters always where separate from the ConstSettings, even in the Settings json file.  This why we did not think about moving it into ConstSettings.